stuf_zds_payments/templates/registrations/contrib/stuf_zds_payments/payment_status_update_xml.txt

```diff
@@ -0,0 +1,8 @@
+{% load capture_tags %}
+{% capture as tt_openvariable silent %}{% templatetag openvariable %}{% endcapture %}
+{% capture as tt_closevariable silent %}{% templatetag closevariable %}{% endcapture %}
+<StUF:extraElementen>
+    <StUF:extraElement naam="payment_completed">{{ tt_openvariable }} payment_completed {{ tt_closevariable }}</StUF:extraElement>
+    <StUF:extraElement naam="payment_amount">{{ tt_openvariable }} payment_amount {{ tt_closevariable }}</StUF:extraElement>
+    <StUF:extraElement naam="payment_public_order_ids">{{ tt_openvariable }} payment_public_order_ids {{ tt_closevariable }}</StUF:extraElement>
+</StUF:extraElementen>
```

Comment:
@sergei-maertens I expected to see here one more <StUF:extraElement> containing the PaymentID from ogone

Reply:
good catch - @stevenbal this is also something that we need to address still. Okay to do this in another PR, since we'll need to update our payment plugins for this too and I'd like to treat that as a separate item/topic.

Reply:
created a separate PR for this https://github.com/open-formulieren/open-forms-ext-stuf-zds-payments/pull/3